manifests/params.pp

```diff
@@ -23,13 +23,22 @@
       $tmp_dir         = '/tmp'
       $install_dir     = undef
       $download_url    = undef
+      case $::osfamily {
+        'RedHat': {
+          $service_provider = 'redhat'
+        }
```

Comment:
Yes it does. I had actually updated the original comment to add that. This is the default provider in < 7:

https://docs.puppetlabs.com/puppet/latest/reference/type.html#service-provider-redhat